glide.lock

```diff
@@ -1,134 +1,36 @@
 hash: b3ffe082966570570810f1f52c796da3dc31d24c84248da9aed9e06db26996ff
-updated: 2017-11-20T13:53:50.869715462-05:00
+updated: 2018-02-01T12:15:16.433775948+02:00
```

Comment:
glide.lock shouldn't need to be updated unless glide.yaml changed.  Did you have to update glide.lock for some reason?

Reply:
reverted